.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8-jdk
```

Comment:
I'm about to check in changes that will let us build with JDK 8. You may want to wait for that.